torchvision/transforms/__init__.py

```diff
@@ -0,0 +1 @@
+from .transforms import *
```

Comment:
Hey, this will import everything in .transforms including its imports. 
You can probably use __all__ variable in transforms.py file. Check models code for an example: https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py#L6

Reply:
ah yes, you're correct. Thanks @chsasank

Reply:
This is now added